mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -84,12 +89,33 @@ func resourceBigtableGCPolicy() *schema.Resource {
 				Description: `The name of the column family.`,
 			},
 
+			"gc_rules": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   `Serialized JSON string for garbage collection policy. Conflicts with "mode", "max_age" and "max_version".`,
+				ValidateFunc:  validation.StringIsJSON,
+				ConflictsWith: []string{"mode", "max_age", "max_version"},
+				StateFunc: func(v interface{}) string {
+					json, _ := structure.NormalizeJsonString(v)
+					return json
+				},
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					var oldJSON map[string]interface{}
+					var newJSON map[string]interface{}
+
+					_ = json.Unmarshal([]byte(old), &oldJSON)
+					_ = json.Unmarshal([]byte(new), &newJSON)
+
+					return reflect.DeepEqual(oldJSON, newJSON)
```

Comment:
Do we have a test cases that test if two nested gc rules are deep equal?

Reply:
As we have synced offline - will delete the DiffSuppressFunc and update design docs.